app/javascript/pages/map/menu/components/sections/datasets/component.jsx

```diff
@@ -1,20 +1,75 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import isEmpty from 'lodash/isEmpty';
+import remove from 'lodash/remove';
 
 import NoContent from 'components/ui/no-content';
 
 import LayerToggle from 'components/map/components/legend/components/layer-toggle';
 import MenuBlock from 'pages/map/menu/components/menu-block';
+import Pill from 'components/ui/pill';
+import Dropdown from 'components/ui/dropdown';
 
 import './styles.scss';
 
 class Datasets extends PureComponent {
   render() {
-    const { datasets, subCategories, onToggleLayer, onInfoClick } = this.props;
+    const {
+      datasets,
+      subCategories,
+      onToggleLayer,
+      onInfoClick,
+      countries,
+      selectedCountries,
+      setMenuSettings
+    } = this.props;
 
     return (
       <div className="c-datasets">
+        <div className="countries-selection">
+          <span className="sub-title">country-specific data</span>
+          <div className="pills">
+            {selectedCountries &&
+              selectedCountries.map(c => (
+                <Pill
+                  key={c.value}
+                  label={c.label}
+                  onRemove={() => {
+                    const newCountries = remove(
+                      selectedCountries,
+                      sc => sc.value !== c.value
```

Comment:
why not use `selectedCountries.filter` instead?

Reply:
Because I am a fool.